src/components/WorkoutCard/WorkoutCard.spec.js

```diff
@@ -1,8 +1,9 @@
 import WorkoutCard from './WorkoutCard'
 import { render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min'
```

Comment:
Should work like this.
```suggestion
import { MemoryRouter } from 'react-router-dom'
```